sdk/rust/oak/src/lib.rs

```diff
@@ -517,8 +518,8 @@ pub fn run_event_loop<T: crate::io::Decodable, N: Node<T>>(mut node: N, in_handl
 
 /// Register a new Node entrypoint.
 ///
-/// This registers the entrypoint name and the expression used to construct the
-/// Node instance. The returned object should implement the [`Node`](trait.Node.html) trait.
+/// This registers the entrypoint name and the expression used to construct the Node instance.
+/// The returned object should be a function that runs [`run_event_loop`].
```

Comment:
These 2 sentences need a reword for the new way of doing things (there's nothing returned, and in particular not a `Node` instance returned).